packages/runtime/src/components/builtin/Video/Video.tsx

```diff
@@ -1,35 +1,31 @@
 import { forwardRef, Ref } from 'react'
 import styled from 'styled-components'
 import ReactPlayer from 'react-player'
+import { cx } from '@emotion/css'
+
 import {
   BorderRadiusValue,
   ElementIDValue,
   MarginValue,
   VideoValue,
-  WidthValue,
 } from '../../../prop-controllers/descriptors'
-import { cssBorderRadius, cssMargin, cssWidth } from '../../utils/cssMediaRules'
+import { cssBorderRadius, cssMargin } from '../../utils/cssMediaRules'
 import { placeholders } from '../../utils/placeholders'
 import { ReactRuntime } from '../../../react'
 import { Props } from '../../../prop-controllers'
 
 type Props = {
   id?: ElementIDValue
   video?: VideoValue
-  width?: WidthValue
+  width?: string
   margin?: MarginValue
   borderRadius?: BorderRadiusValue
 }
 
-const Container = styled.div<{
-  width: Props['width']
-  margin: Props['margin']
-  borderRadius: Props['borderRadius']
-}>`
+const Container = styled.div<{ margin: Props['margin']; borderRadius: Props['borderRadius'] }>`
   display: flex;
   flex-direction: column;
   overflow: hidden;
-  ${cssWidth('560px')}
```

Comment:
Where did this `560px` go?

Reply:
https://github.com/makeswift/makeswift/pull/39#discussion_r852432673